velox/exec/TopNRowNumber.h

```diff
@@ -84,6 +84,10 @@ class TopNRowNumber : public Operator {
     std::priority_queue<char*, std::vector<char*, StlAllocator<char*>>, Compare>
         rows;
 
+    // This is the highest rank seen so far in the input rows. It is compared
+    // with the limit for the operator.
+    int64_t topRank = 0;
```

Comment:
`topRank `-> `topNRowNumbers`?

Reply:
I'm trying to update this code so that there are few diffs when enhancing for rank functions. So named accordingly. I'll revise this comment indicating that.